sdk/storage/azure-storage-blob-changefeed/setup.py

```diff
@@ -77,6 +77,6 @@
     },
     python_requires=">=3.8",
     install_requires=[
-        "azure-storage-blob>=12.14.1,<13.0.0"
+        "azure-storage-blob>=12.19.0,<13.0.0"
```

Comment:
Can you add an entry to the changefeed changelog saying we bumped the blob dependency to this?